test/draw/gpu/BlitTest.ooc

```diff
@@ -91,17 +91,73 @@ BlitTest: class extends Fixture {
 			this testOffset(this sourceImage, correctImage, IntVector2D new(4, 1))
 			correctImage free()
 		})
+		this add("Blit blend", func {
+			correctImage := RasterMonochrome fromAscii(
+				"< *X>
+				<XXXXXXXX       >
+				<X*    *X       >
+				<X*    *X       >
+				<X*   XXX       >
+				<X****XXXXXXXX  >
+				<XXXXXXXX   *X  >
+				<     X*    *X  >
+				<     X*   XXX  >
+				<     X****XXX  >
+				<     XXXXXXXX  >
+				<               >
+				<               >"
```

Comment:
This is valid for both white and additive blending because they second sprite only overlap with black and white pixels from the first sprite.